presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -6336,6 +6336,64 @@ public void testAnalyzeUnpartitionedTable()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testAnalyzeTableTimestampWithPrecision()
+    {
+        String catalog = getSession().getCatalog().get();
+        Session nanosecondsTimestamp = Session.builder(withTimestampPrecision(getSession(), HiveTimestampPrecision.NANOSECONDS))
+                // Disable column statistics collection when creating the table
+                .setCatalogSessionProperty(catalog, "collect_column_statistics_on_write", "false")
+                .build();
+        Session microsecondsTimestamp = withTimestampPrecision(getSession(), HiveTimestampPrecision.MICROSECONDS);
+        Session millisecondsTimestamp = getSession();
```

Comment:
How do you  know millis is the default?
Maybe call `withTimestampPrecision` here too?
or verify